mitiq/rem/inverse_confusion_matrix.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (C) 2022 Unitary Fund
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from typing import List
+import numpy as np
+import numpy.typing as npt
+
+from mitiq._typing import MeasurementResult, Bitstring
+
+
+def sample_probability_vector(
+    probability_vector: npt.NDArray[np.float64], samples: int
+) -> npt.NDArray[np.uint8]:
```

Comment:
Should this function returns `Bitstring`s?

Reply:
Yes, that makes more sense since I was calling `.tolist()` where it was used.